presto/dataset.py

```diff
@@ -219,6 +229,13 @@ def join_with_world_df(dataframe: pd.DataFrame) -> pd.DataFrame:
         return joined.to_crs("EPSG:4326")
 
 
+class WorldCerealLabelledMaizeDataset(WorldCerealLabelledDataset):
+    @staticmethod
+    def target_from_row_dict(row_d) -> int:
```

Comment:
To be more closely aligned with the hierarchic nature of WorldCereal (first map out crops, then map the crops inside cropland), we should think about how to do that.

in principle `row_d["LANDCOVER_LABEL"] != 11` should indicate which rows are not to be used to train the crop type models. Or maybe  `row_d["LANDCOVER_LABEL"] not in [10, 11, 12, 13]` to also have perennials and grassland in to be sure. I guess Presto can then also focus entirely on the crops themselves?

Is this filtering mechanism something we should do when initializing the dataset with a dataframe?

Reply:
Added to the eval task